examples/token-vendor/src/lib.rs

```diff
@@ -0,0 +1,229 @@
+#![no_std]
```

Comment:
None previous issues wasn't resolved. They are pretty similar to `contract-example-reply`'s issues.